Jenkinsfile

```diff
@@ -48,7 +48,7 @@ pipeline {
         stage('Checkout repo') {
             steps {
                 retry(3) {
-                    checkout scm
+                    checkout([$class: 'GitSCM', branches: [[name: "refs/heads/${BRANCH_NAME}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'WipeWorkspace'], [$class: 'LocalBranch', localBranch: "${BRANCH_NAME}"], [$class: 'CloneOption', timeout: 30, shallow: true]], submoduleCfg: [], userRemoteConfigs: scm.userRemoteConfigs ])
```

Comment:
❔ Are we setting the value of BRANCH_NAME on the job configuration?

Reply:
It's a preset global variable as part of the Multi branch pipeline build.

Reply:
Great, thanks for the explanation 👍